### PR TITLE
Update docs for registry v2.8.0 release

### PR DESCRIPTION
**Pull request by milosgajdos**

### CODE REVIEW

File: registry/get-help.md [hunk 1]
```diff
@@ -1,3 +1 @@
-[the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](https://dockr.ly/slack), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker)
-
--	**Need a free, secure hosted registry?** Try [Docker Hub](https://www.docker.com/pricing).
```
Comment (tianon) [inline registry/get-help.md:3]: This was added over in #1798 in conversation with @j0hnst0n - I guess now that Distribution is CNCF, this is a bit more complicated. :grimacing:

(Apologies if a conversation around this already happened and the removal was intentional/pre-discussed!)

**Review by tianon — COMMENTED** [review]
Nice!  Great to see a logo. :+1:

Overall, this looks good - just a couple minor comments / discussion points.

FWIW, `maintainer.md` is intended to be the _image_ maintainers, not the software maintainers, so I would suggest we wait to change that until we work out where the final home of the image repository is :see_no_evil: (for some reason GitHub's having a glitch and won't let me comment on the line from that file directly)

File: registry/get-help.md (hunk 1 above)
Comment (milosgajdos) [reply registry/get-help.md:3]: There was a discussion about all removing references to Docker (Hub) from all docs now that distribution has been donated to CNCF, yeah. Hence this change.

**Comment by milosgajdos** [comment]
> FWIW, `maintainer.md` is intended to be the _image_ maintainers, not the software maintainers

We basically followed what `fluentd` have done, but I see what you mean. Yeah, we need to decide where the official image repo lives before we make this change.

File: registry/content.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Docker Registry
```
Comment (thaJeztah) [inline registry/content.md:1]: This probably should be changed to something else

I _think_ this was part of the reason for adding https://github.com/docker-library/docs/pull/2113/files#r801121627 (https://github.com/docker-library/docs/pull/1798), as users tended to confuse `Docker Registry` and `Docker Hub (Registry)`.

Comment (milosgajdos) [reply registry/content.md:1]: Should it be simply just `Distribution`?

File: registry/content.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
 
-This image contains an implementation of the Docker Registry HTTP API V2 for use with Docker 1.6+. See [github.com/docker/distribution](https://github.com/docker/distribution) for more details about what it is.
+%%LOGO%%
+
+This image contains an implementation of the Docker Registry HTTP API V2 for use with Docker 1.6+. See [github.com/distribution/distribution](https://github.com/distribution/distribution) for more details about what it is.
```
[the remark this answers is not in the document]
Comment (milosgajdos) [reply registry/content.md:5]: I think mentioning V2 is still correct -- especially for this release! But we can certainly add the mention of OCI, though I'm not sure how compliant v2.8.0 with OCI is...`main` branch is fully compliant, though.

File: registry/get-help.md (hunk 1 above)
Comment (milosgajdos) [reply registry/get-help.md:3]: https://github.com/docker-library/docs/pull/2113/commits/ffb8ea0a91cd22a64143cd75cb151de3c8a0a1db

**Comment by milosgajdos** [comment]
> FWIW, maintainer.md is intended to be the image maintainers, not the software maintainers, so I would suggest we wait to change that until we work out where the final home of the image repository is 🙈

Makes sense, I've put the original ref back: https://github.com/docker-library/docs/pull/2113/commits/3846c5cb1d9d9ad1bb2455c07e1ea2c085fe836d

File: registry/content.md (hunk 2 above)
Comment (tianon) [reply registry/content.md:1]: Should we merge for now with this as-is, or do you want to resolve this before merge?

Comment (milosgajdos) [reply registry/content.md:1]: let me rename it to `Distribution`, @tianon, since we now have this shiny new logo :)

**Pull request merged**